lib/nlp/nlp-manager.js

```diff
@@ -442,11 +442,54 @@ class NlpManager {
   }
 
   /**
-   * Save the NLP manager information into a file.
-   * @param {String} srcFileName Filename for saving the NLP manager.
+   * Load NLP manager information from a string.
+   * @param {String} data JSON string to load NLP manager information from.
    */
-  save(srcFileName) {
-    const fileName = srcFileName || 'model.nlp';
+  import(data) {
+    const clone = JSON.parse(data);
```

Comment:
If here we do a typeof check, we can have import working with incoming strings, that will be parsed, or objects.
```javascript
const clone = typeof data === 'string' ? JSON.parse(data) : data;
```

Reply:
Sounds good! Will do.